src/app/static/src/app/components/modelOutput/ModelOutput.vue

```diff
@@ -100,6 +100,7 @@
                     :formatFunction="formatBarchartValue"
                     :showRangesInTooltips="true"
                     @update="updateComparisonPlotSelectionsAndXAxisOrder"></bar-chart-with-filters>
+                <error-alert v-if="hasComparisonPlotError" :error="comparisonPlotError"></error-alert>
```

Comment:
```suggestion
                <error-alert v-if="!!comparisonPlotError" :error="comparisonPlotError"></error-alert>
```

Reply:
I can change this, but this is the same pattern used in `modelOptions.vue`